simde/x86/sse4.1.h

```diff
@@ -1475,15 +1475,15 @@ simde_mm_insert_ps (simde__m128 a, simde__m128 b, const int imm8)
     r_,
     a_ = simde__m128_to_private(a),
     b_ = simde__m128_to_private(b);
-
-  a_.f32[0] = b_.f32[(imm8 >> 6) & 3];
-  a_.f32[(imm8 >> 4) & 3] = a_.f32[0];
-
+ 
+  float tmp1_ = b_.f32[(imm8 >> 6) & 3];
+  a_.f32[(imm8 >> 4) & 3] = tmp1_;
+  
   SIMDE_VECTORIZE
   for (size_t i = 0 ; i < (sizeof(r_.f32) / sizeof(r_.f32[0])) ; i++) {
-    r_.f32[i] = (imm8 >> i) ? SIMDE_FLOAT32_C(0.0) : a_.f32[i];
+    r_.f32[i] = ((imm8 >> i) & 1 ) ? SIMDE_FLOAT32_C(0.0) : a_.f32[i];
   }
-
+  
```

Comment:
Can you remove the trailing whitespace?

https://github.com/simd-everywhere/simde/runs/4104134014?check_suite_focus=true#step:4:4